client/src/components/ui/form.tsx

```diff
@@ -0,0 +1,278 @@
+import * as LabelPrimitive from "@radix-ui/react-label";
+import { Slot } from "@radix-ui/react-slot";
+import React from "react";
```

Comment:
This affects how nextjs tree shakes the imports. Use `import * as React from "react";` instead or remove it because nextjs imports it already